operator/docs/lokistack/object_storage.md

```diff
@@ -152,35 +152,82 @@ Loki Operator supports [AWS S3](https://aws.amazon.com/), [Azure](https://azure.
 
-* Create an Object Storage secret with keys as follows:
+  ```yaml
+    apiVersion: objectbucket.io/v1alpha1
+    kind: ObjectBucketClaim
+    metadata:
+      name: loki-bucket-odf
+      namespace: openshift-logging
+    spec:
+      generateBucketName: loki-bucket-odf
+  ```
 
-    ```console
-    kubectl create secret generic lokistack-dev-odf \
-      --from-literal=bucketnames="<BUCKET_NAME>" \
-      --from-literal=endpoint="https://s3.openshift-storage.svc" \
-      --from-literal=access_key_id="<ACCESS_KEY_ID>" \
-      --from-literal=access_key_secret="<ACCESS_KEY_SECRET>"
-    ```
+* Get bucket properties from the associated ConfigMap:
+  ```console
+  BUCKET_HOST=$(oc get -n openshift-logging configmap loki-bucket-odf -o jsonpath='{.data.BUCKET_HOST}')
+  BUCKET_NAME=$(oc get -n openshift-logging configmap loki-bucket-odf -o jsonpath='{.data.BUCKET_NAME}')
+  BUCKET_PORT=$(oc get -n openshift-logging configmap loki-bucket-odf -o jsonpath='{.data.BUCKET_PORT}')
+  ```
 
-    where `lokistack-dev-odf` is the secret name. You can copy the values for `BUCKET_NAME`, `ACCESS_KEY_ID` and `ACCESS_KEY_SECRET` from your ObjectBucketClaim's accompanied secret.
+* Get bucket access key from the associated Secret:
+  ```console
+  ACCESS_KEY_ID=$(oc get -n openshift-logging secret loki-bucket-odf -o jsonpath='{.data.AWS_ACCESS_KEY_ID}' | base64 -d)
+  SECRET_ACCESS_KEY=$(oc get -n openshift-logging secret loki-bucket-odf -o jsonpath='{.data.AWS_SECRET_ACCESS_KEY}' | base64 -d)
+  ```
+
+  * Create an Object Storage secret with keys as follows:
+
+  ```console
+    oc create -n openshift-logging secret generic lokistack-dev-odf \
+    --from-literal=access_key_id="${ACCESS_KEY_ID}" \
+    --from-literal=access_key_secret="${SECRET_ACCESS_KEY}" \
+    --from-literal=bucketnames="${BUCKET_NAME}" \
+    --from-literal=endpoint="https://${BUCKET_HOST}:${BUCKET_PORT}"
+  ```
+
+  Where `lokistack-dev-odf` is the secret name. The values for `ACCESS_KEY_ID`, `SECRET_ACCESS_KEY`, `BUCKET_NAME`, `BUCKET_HOST` and `BUCKET_PORT` are taken from your ObjectBucketClaim's accompanied secret and ConfigMap.
 
 * Create an instance of [LokiStack](../hack/lokistack_dev.yaml) by referencing the secret name and type as `s3`:
 
   ```yaml
+  apiVersion: loki.grafana.com/v1
+  kind: LokiStack
+  metadata:
+    name: logging-loki
+    namespace: openshift-logging
   spec:
     storage:
       secret:
         name: lokistack-dev-odf
         type: s3
+    tls:
+      caName: openshift-service-ca.crt
+  tenants:
+    mode: openshift-logging
+  ```
+
+* Create a ClusterLogging instance in the `openshift-logging` namespace referencing the LokiStack above:
+  ```yaml
+  apiVersion: logging.openshift.io/v1
+  kind: ClusterLogging
+  metadata:
+    name: instance
+    namespace: openshift-logging
+  spec:
+    collection:
+      type: vector
+    logStore:
+      lokistack:
+        name: logging-loki
+      type: lokistack
```

Comment:
Although the general case for OpenShift Logging on ODF users will be on OpenShift, I would skip adding this here as it is clearly OpenShift docs related. The Loki Operator Docs are related only to LokiStack.

Reply:
Understood.
In order for users to get the complete picture when using OpenShift, at least we can mention that a creation of `ClusterLogging` CR is required and maybe link to step 4 [here](https://docs.openshift.com/container-platform/4.12/logging/config/cluster-logging-configuring-cr.html)?

Reply:
Also, do you think we should stick with `kubectl`, or use `oc` since it is being done on an OpenShift cluster?

Reply:
> Also, do you think we should stick with `kubectl`, or use `oc` since it is being done on an OpenShift cluster?

We can stick to `kubectl` as per these docs are upstream only.

Reply:
What about adding a comment like:
> Configure a logging collector that references the LokiStack.

So users will know they have one additional step to be able to view the logs

Reply:
Sure thing sounds as a good generic note for the top of this page (regardless ODF).

Reply:
Added a node on the top and updated the PR.